eng/pipelines/runtime.yml

```diff
@@ -1,4 +1,27 @@
-trigger: none
+# Setting batch to true, triggers one build at a time.
+# if there is a push while a build in progress, it will wait,
+# until the running build finishes, and produce a build with all the changes
+# that happened during the last build.
+trigger:
+  batch: true
+  branches:
+    include:
+    - master
+    - release/*.*
+  paths:
+    include:
+    - '*'
+    - docs/manpages/*
+    exclude:
+    - eng/Version.Details.xml
```

Comment:
just for consistency, can you please also add this in the pr section below.

Reply:
Yeah, I will also add it to the path detection under the checkout job filters.